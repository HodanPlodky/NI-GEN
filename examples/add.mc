int f() {
    return 1 + 3;
}

int main() {
    return 11 + 2;
}
