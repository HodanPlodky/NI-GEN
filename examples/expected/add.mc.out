input file : examples/tests/add.mc
return 21
